rosplan_knowledge_base/include/rosplan_knowledge_base/VALVisitorProblem.h

```diff
@@ -54,18 +57,24 @@ namespace KCL_rosplan {
 		rosplan_knowledge_msgs::DomainFormula last_func_term;
 
 		std::map<std::string, std::vector<std::string> > instances;
+		std::map<std::string, std::vector<std::string> > constants;
```

Comment:
Currently there are no constants parsed in the problem parser - we will have to add this.